gateways/Todo.gateway.ts

```diff
@@ -0,0 +1,6 @@
+import AsyncStorage from "@react-native-async-storage/async-storage";
+
+export async function loadFromStorage() {
+  const serializedTodos = await AsyncStorage.getItem("todos");
```

Comment:
they key can be a constant.
And in general these type of constants is good to have them in a single config-like file.
So if in future you wish to change their values, you don't have to chase them all over the code ;-)